updated VimLab

yaj030 opened the pull request:
add run line, run selection, run file, clear screen, clear all, add breakpoint

yaj030 committed:
matlab_vimlab.vim

yaj030 committed:
Delete matlab_vimlab.m

yaj030 committed:
Update matlab_vimlab.vim

yaj030 committed:
Update VimLab. 

add quick fix list feature to look at the errors. Basically it create a temporary .m file called MatVimTmp.m in the same folder and run it and then catch the errors which are saved in the /tmp/MatlabErrorForVim.err. when press \mq, the errors are loaded by calling cfile('/tmp/MatlabErrorForVim.err')

yaj030 committed:
Update readme

yaj030 committed:
fix the delete file bug, delete the error file

yaj030 committed:
add redraw and silent

jeroendehaas commented:
I've recently been very occupied with changes in real life. I am going to work on this soon and integrate it.

yaj030 committed:
revise the RunMatFileRevise function 

consider the case that some m file are function files, cann't use try .. function

yaj030 committed:
test git commands

yaj030 committed:
delete the clc when start up, cause prob when quit
